main.go

```diff
@@ -97,12 +97,19 @@ func (m *MozProcessStat) Diff(data MozProcessStat) {
 	m.NetworkIO.Fifoout -= data.NetworkIO.Fifoout
 }
 
+// SystemMemoryInfo summarises information about the system memory usage
+type SystemMemoryInfo struct {
+	TotalMemory      uint64  `json:"vmem_total`
+	TotalSwap        uint64  `json:"swap_total"`
+	AvailableMemory  uint64  `json:"vmem_available`
```

Comment:
I think this needs to be collected at every sample rather than just the once. It's 'available' as in 'Total in system minus total in use minus overhead from OS' rather than an unchanging value